types/vis/index.d.ts

```diff
@@ -160,7 +160,7 @@ export interface TimelineOptions {
   itemsAlwaysDraggable?: boolean;
   locale?: string;
   locales?: any; // TODO
-  moment?(): void; // TODO
+  moment?(date: Date): Date;
```

Comment:
I think this can be a `Date` or `number` or `string` or `moment`or possibly a function?

Will have to confirm by doing some testing or by discussing with someone who knows the library better.